package.json

```diff
@@ -175,6 +175,7 @@
     "semver": "^7.5.4",
     "word-wrap": "^1.2.4",
     "amplify-codegen": "4.7.3",
-    "@aws-amplify/appsync-modelgen-plugin": "2.8.1"
+    "@aws-amplify/appsync-modelgen-plugin": "2.8.1",
+    "tar": "^6.2.1"
```

Comment:
Via override because `lerna` depends in it and doesn't have fix in `6.x` line (would require MV bump otherwise).